src/app/garfile/submit/success/index.njk

```diff
@@ -13,7 +13,6 @@
 {% block beforeContent %}
 
   {% include "phase.njk" %}
-  {% include "common/templates/includes/back.njk" %}
```

Comment:
Question: Should it be done also for the `app/garfile/submit/failure/index.njk`?
- That currently sets the gar's status to Submitted.
- Therefore, with the back button would send the user to the review page of a "Submitted" (but failed) gar.

Reply:
Not sure, what will be the state of the in case of a failure, if it is NOT in the Submitted state, user need an option to go back to the previous page to review and correct the error.

Reply:
Ok, that is fine.
I thought that when the application submitted a GAR to CBP but failed, it still changed the GAR's status to a Submitted state.
* However, if I am incorrect, then the above is the correct behaviour.